sql_it_test.go

```diff
@@ -390,6 +439,91 @@ func TestSQLWithPortableDateTime(t *testing.T) {
 	})
 }
 
+func TestSQLWithPortableDateTime2(t *testing.T) {
```

Comment:
If this does this bring any extra value over the similar test on "driver/driver_it_test.go", I think we can remove one of them. Wdyt?

Reply:
They are mostly similar but not the same. This one tests the SQL methods on the Client, and the other one tests the driver.